MessengerOnFileForLinux/Common/src/SignalHandling.cpp

```diff
@@ -82,4 +84,46 @@ void createPosixSignalsHandling(void(*handlingFunction)(int))
         std::signal(posixSignal, handlingFunction);
     }
 }
+
+namespace NCurses
```

Comment:
namespace zbyt podobny do ncurses, trzeba to odróżnić od biblioteki

Reply:
niby tak, ale jest wpakowany w SignalHandling::NCurses to wydaje mi się ze nie ma kolizji nazw. Na poczatku miałem NCursesSignal, ale to bez sensu dublowanie Signal z poprzedniego namespace

Reply:
Ale tak samo mógłbyś ncursowy wrzucić do środka swojego. Twoja propozycja nazwy też nie jest do końca okey. Podtrzymuję, że trzeba to wrzucić do klasy.

Reply:
ok